Lib/DocXMLRPCServer.py

```diff
@@ -210,7 +210,15 @@ def generate_html_documentation(self):
                                 methods
                             )
 
-        return documenter.page(self.server_title, documentation)
+        escape_table = {
+            "&": "&amp;",
+            '"': "&quot;",
+            "'": "&#x27;",
+            ">": "&gt;",
+            "<": "&lt;",
+        }
+        title = ''.join(escape_table.get(c, c) for c in self.server_title)
```

Comment:
Please don't inline html.escape() here, but add a private function at top-level. Function name like "_html_escape_quote()" with code copied from Lib/html/__init__.py. You can hardcode quote=True in the function.

I don't think that your code behaves as html.escape() in master. The replace must be done in a specific order. In case of doubt, I prefer to reuse code from master.

Reply:
I 've updated the PR as you comment!
Thanks for the feedback.

> The replace must be done in a specific order

Yes this should be in master version
But b41cde823d026f2adc21ef14b1c2e92b1006de06 version does not  has to care
about ordering.
Because it does not use replace but iterating and changing.

For example
'<&'
on the master version without ordering can be happen in this below scenario
````
s = '<&'
s = '&lt;&' # if < is replaced first
s = '&amp;lt;&amp;' # what?
````

but on the b41cde823d026f2adc21ef14b1c2e92b1006de06 version
````
s = '<&'
l = [&lt;], c ='<' s = '<&'
l = [&lt; &amp;], c ='&' s = '<&'
''.join(l) # will be '&lt; &amp;' we expacted.
````

I'd like to optimize html.escape of master version with b41cde823d026f2adc21ef14b1c2e92b1006de06 
if this showing better performance.
What do you think?

Reply:
I found that it's not worth to do.